pom.xml

```diff
@@ -242,7 +242,7 @@
 		          <dependency>
 		            <groupId>org.codehaus.plexus</groupId>
 		            <artifactId>plexus-compiler-eclipse</artifactId>
-		            <version>2.8.5</version>
+		            <version>2.8.5-spigotmc</version>
```

Comment:
It is actually provided by spigot and i think it is used in spigot, but why it fails the build?